opentelemetry-dynatrace/Cargo.toml

```diff
@@ -0,0 +1,92 @@
+[package]
+name = "opentelemetry-dynatrace"
+version = "0.1.0"
+authors = ["OpenTelemetry Authors <cncf-opentelemetry-contributors@lists.cncf.io>"]
+description = "Dynatrace exporters and propagators for OpenTelemetry"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-dynatrace"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-dynatrace"
+readme = "README.md"
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+    "asynchronous",
+]
+keywords = ["opentelemetry", "metrics", "dynatrace"]
+license = "Apache-2.0"
+edition = "2018"
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[[test]]
+name = "integration_tokio"
+path = "tests/http_test.rs"
+required-features = ["rt-tokio"]
+
+[[test]]
+name = "integration_async_std"
+path = "tests/http_test.rs"
+required-features = ["rt-async-std"]
```

Comment:
I think currently there are only tokio tests in `tests/http_test.rs`?

Reply:
I removed the configuration for `async-std`